client/src/components/Home/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import swal from 'sweetalert';
+import { Link } from 'react-router-dom';
+import { Grid, Box, Typography, Paper, IconButton } from '@material-ui/core';
+import { AddCircleOutline } from '@material-ui/icons';
+import LoaderProgress from '../../common-components/LoaderProgress';
+import Water from '../../assets/water.svg';
+import Communication from '../../assets/communication.svg';
+import Electricity from '../../assets/electricity.svg';
+import Internet from '../../assets/internet.svg';
+import Styles from './style';
+
+export default () => {
+  const classes = Styles();
+  const [userBillType, setUserBillType] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [displayBlock, setIsDisplayBlock] = useState(true);
+
+  useEffect(() => {
+    axios
+      .get('/api/home')
+      .then((result) => {
+        setUserBillType(result.data.Result);
+      })
+      .catch((err) => {
+        if (err.response.data) {
+          swal('Error', err.response.data.message, 'error');
+        }
+        setIsLoading(false);
+      });
+  }, []);
+
+  const BillCard = (index, type, imgSrc) => (
+    <Link to={`/bill/${type}`}>
+      <Paper key={index} className={classes.root} elevation={3}>
+        <img src={imgSrc} className={classes.imageType} />
+      </Paper>
+    </Link>
+  );
+
+  const buildBillType = (types) => {
+    if (types.length === 0) {
```

Comment:
if the user doesn't have any bill, the loading still working, So as a user I need to see the result after the loading you can write a message to tell the user to add bills.